walletconnect/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "vanillajs-walletconnect",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "vite": "^5.4.0"
+  },
+  "dependencies": {
+    "@hashgraph/hedera-wallet-connect": "1.3.2",
+    "@hashgraph/sdk": "2.49.2",
+    "@walletconnect/modal": "2.6.2",
+    "ethers": "5.7.2"
```

Comment:
try removing the `ethers` dependency, as I do not see it being imported in the main JS file. it may be necessary (e.g. if one of the other dependencies defines it as a peer dependency).

likewise for `@walletconnect/modal`.